bazel/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("@aspect_bazel_lib//lib:write_source_files.bzl", "write_source_files")
```

Comment:
@gregmagolan I think these all need to be dropped, the copybara config isn't dropping them atm

Reply:
I can take another pass on getting this "copybara-clean".

We had a colliding vision on how to do `.pb.go` file generation between the two repos :(

Reply:
I'll take over this PR...